docs/config.json

```diff
@@ -2168,11 +2168,6 @@
       "destination": "/access-controls/access-request-plugins/ssh-approval-jira/",
       "permanent": true
     },
-    {
-      "source": "/enterprise/workflow/ssh-approval-jira-cloud/",
-      "destination": "/access-controls/access-request-plugins/ssh-approval-jira/",
```

Comment:
Is there somewhere new we could point the redirects to, or do those pages just don't exist in any way anymore?

Reply:
I'm not sure why this redirect is being deleted here, but in general, we are reworking our process for creating redirects this month, so I am okay with changing some here.

Would you be able to provide context, @iAdramelk?

Reply:
@ptgott This redirect was removed because it was an exact copy of the previous one (see lines 2167-2170). There were a couple of duplicate redirects in the file. It was not an error for the old docs engine, but Docusaurus is throwing an error on them.

Reply:
Makes sense, thanks.